WordPress/wporg-two-factor
# [WIP] Add dynamic TOTP functionality

Fixes #30 

This still has some rough edges, but works pretty well. We'll definitely need to iterate on the progress bar design, but I think it'll be helpful to get the functionality merged and then circle back to that.

To test:

1. disable TOTP via `wp-admin/profile.php`
2. visit `/users/{username}/edit/account/`. the status should show that it's disabled.
3. click on totp screen and follow process to setup using qr code
4. repeat steps using text code instead of qr code
5. repeat steps using keyboard instead of mouse
